### ISSUE TITLE
example: xml

### ISSUE THREAD

**Issue by vbrazo** [v-community/v_by_example#53]

**Comment by vbrazo** [comment]
We should probably consider writing about this library https://github.com/zenith391/vwxml.

**Comment by dhonx** [comment]
@vbrazo Yeah... I will love to do this, he is my friend. Or maybe @zenith391 will help us to do this?
